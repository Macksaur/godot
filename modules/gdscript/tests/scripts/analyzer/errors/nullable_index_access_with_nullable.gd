func test():
    var foo: String? = null
    var nullable_index: int? = null
    foo[nullable_index]