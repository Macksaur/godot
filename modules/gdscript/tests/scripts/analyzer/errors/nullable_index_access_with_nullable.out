GDTEST_ANALYZER_ERROR
Potentially unsafe to use nullable "nullable_index" as an index.
